examples/image_classifier/resnet_152_batch/resnet152_handler.py

```diff
@@ -1,38 +0,0 @@
-import torch
-
```

Comment:
Deleting this example?

Reply:
Yes. The native handlers can now handle batching, so this extra handler is unnecessary.

Reply:
Can update associated doc https://github.com/pytorch/serve/blob/master/docs/batch_inference_with_ts.md